src/snovault/local_storage.py

```diff
@@ -0,0 +1,133 @@
+import binascii
+
+from os import urandom
+from datetime import datetime
+from pytz import timezone
+
+from redis import StrictRedis
+from pyramid.view import view_config
+
+from snovault.interfaces import (
+    LOCAL_STORAGE,
+    LOCAL_STORAGE_EVENT_TAGS,
+    LOCAL_STORAGE_STATE,
+)
+
+
+def includeme(config):
+    config.scan(__name__)
+    config.add_route('local_storage', '/local_storage')
+    config.registry[LOCAL_STORAGE] = LocalStoreClient(
+        db=config.registry.settings['local_storage_db'],
+        host=config.registry.settings['local_storage_host'],
+        port=config.registry.settings['local_storage_port'],
+        timeout=int(config.registry.settings['local_storage_timeout']),
+    )
+
+
+@view_config(route_name='local_storage', request_method='GET')
+def local_storage(request):
+    '''Examples of how to store and get from redis'''
+    local_store = request.registry[LOCAL_STORAGE]
+    local_tz = request.registry.settings.get('local_tz')
+    result = {
+        'utc_now': str(datetime.utcnow()),
+        'lcl_now': f"{local_tz}: {datetime.now(timezone(local_tz))}",
+        # Different pyramid process may have different registries, this returns memory address
+        'local_storage': str(local_store),
+    }
+
+    # Set/Get dict or hash
+    redis_info = local_store.dict_get(LOCAL_STORAGE_STATE)
+    if not redis_info:
+        redis_info = {
```

Comment:
should this be part of the LocalStoreClient initializer, so we always have a state ready?

Reply:
That's a good idea.  It's more static that way.  Also on initialization the connection will be checked.  Connection errors don't happen until a query is made.